src/app/menu/menu.component.html

```diff
@@ -0,0 +1,106 @@
+<p class="title">Main Menu</p>
+<div class="menu">
+    <div class="col-lg-5 col-lg-offset-4">
+        <div class="row">
+            <div class="menu-button">
+                <img class="hex" src="../../assets/img/intro.png">
```

Comment:
Yes I did. The images looked better in comparison. Also there are icons inside the hexagon for which I will have to use those icon images.